systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/controllers/JobUtils.java

```diff
@@ -43,13 +43,12 @@ public static void deleteJobWithWait(String namespace, String name) {
     /**
      * Wait for specific Job failure
      * @param jobName job name
-     * @param namespace namespace
      * @param timeout timeout in ms after which we assume that job failed
      */
     public static void waitForJobFailure(String jobName, String namespace, long timeout) {
         LOGGER.info("Waiting for job: {} will be in error state", jobName);
         TestUtils.waitFor("job finished", Constants.GLOBAL_POLL_INTERVAL, timeout,
-            () -> !kubeClient().checkSucceededJobStatus(jobName));
+            () -> !kubeClient().checkSucceededJobStatus(jobName, namespace, 1));
```

Comment:
```    
public Boolean checkSucceededJobStatus(String namespaceName, String jobName, int expectedSucceededPods) {
        return getJobStatus(namespaceName, jobName).getSucceeded().equals(expectedSucceededPods);
    }
```
    
    it looks to me that parameters order is wrong. How is possible that tests are green?

Reply:
Good catch! I have change it in one test as well.